Block/Checkout/Multishipping/Success.php

```diff
@@ -95,6 +103,8 @@ public function __construct(
 
         $orderIds = $this->getOrderIds();
 
+        $this->isAdyenPayment = false;
```

Comment:
Maybe this isn't necessary? The initial value of the property would be null, which I think would be returned as false by `isAdyenPayment(): bool`

Reply:
True that, thank you! But then a type error was thrown for `null` being returned, so I also changed the `isAdyenPayment(): bool` to `isAdyenPayment(): ?bool`